src/sentry/mediators/sentry_apps/updater.py

```diff
@@ -93,6 +95,14 @@ def _update_redirect_url(self):
     def _update_is_alertable(self):
         self.sentry_app.is_alertable = self.is_alertable
 
+    @if_param('verify_install')
+    def _update_verify_install(self):
+        if self.sentry_app.is_internal:
+            raise APIError(
+                u'Cannot update this option for internal integrations.',
```

Comment:
What about something like "Internal Integrations do not require installation verification"? It words it a little more around the reason why this is invalid, instead of just saying it's invalid, imo.